### PR TITLE
Create free-courses-zh.md

### PR DESCRIPTION
**Pull request by GH05T-HUNTER5**
Chinese Tutorial 🤔

Check yourself 😉

If there is any error in this please close it

(Close This free-courses-zh.md)

### CODE REVIEW

**Comment by davorpa** [comment]
### Review Note

To be Merged jointly, one following by the other, with #7014

> Chinese Tutorial 🤔
> 
> Check yourself 😉
> 
> If there is any error in this please close it
> 
> (Close This free-courses-zh.md)

... and of course close related PRs

**Comment by davorpa** [comment]
### Review Note

To be Merged jointly, one following by the other, with #7014

> Chinese Tutorial 🤔
> 
> Check yourself 😉
> 
> If there is any error in this please close it
> 
> (Close This free-courses-zh.md)

... and of course close related PRs

**Comment by davorpa** [comment]
### Review Note

To be Merged jointly, one following by the other, with #7014

> Chinese Tutorial 🤔
> 
> Check yourself 😉
> 
> If there is any error in this please close it
> 
> (Close This free-courses-zh.md)

... and of course close related PRs

File: courses/free-courses-zh.md [hunk 1]
```diff
@@ -27,6 +35,14 @@
 * [【尚硅谷】JavaScript教程视频140集实战教学,从入门到精通 2019适合零基础by李立超 【提供源码笔记】](https://youtube.com/playlist?list=PLmOn9nNkQxJFubqN777c_nScnJ4dpEYMT) - 尚硅谷IT培训学校
 
 
+#### Node.js
+
+* [🧶NodeJS中文入門教學課程 (廣東話/粵語)](https://youtube.com/playlist?list=PL91gvBcdjUtZVtROM5MY-DTlk4pfnqbm_) - 梁浩賢
```
Comment (davorpa) [inline courses/free-courses-zh.md:40]: Read CONTRIBUTING.MD

> - No emojis.

File: courses/free-courses-zh.md [hunk 2]
```diff
@@ -27,6 +35,14 @@
...
+
+
+#### React.js
+
+* [React系列項目從零基礎到項目開 發實戰](https://youtube.com/playlist?list=PL9nxfq1tlKKnza3MPogWqaYIPtdW_G2lF) - Jomy King
```
Comment (davorpa) [inline courses/free-courses-zh.md:45]: 2 blank lines need be guarded after each listing  and just next section title (see linter report). Here are zero

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-zh.md (hunk 2 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-zh.md:45]: Ok

File: courses/free-courses-zh.md (hunk 1 above)
Comment (GH05T-HUNTER5) [reply courses/free-courses-zh.md:40]: Ok

**Comment by eshellman** [comment]
1. Please read https://www.loc.gov/standards/iso639-2/faq.html#24  At least one of these courses is in cantonese, and we need to decide how to handle Mandarin vs. Cantonese (not an issue for _written_ resources.
2. Please close #7014 and include the changes there with this PR. As we've seen with other new lists, having two PRs is very error prone.
3. I will not merge this PR without feedback from Mandarin/Cantonese speakers.

**Comment by GH05T-HUNTER5** [comment]
> 

Let me close these two @eshellman

**Comment by GH05T-HUNTER5** [comment]
I'm closing it because I can't add Chinese here

**Pull request closed without merging**